### DISCUSSION TITLE
variable timeout when WiFi is lost to starting HOTSPOT?

### DISCUSSION THREAD

**Discussion by mennamorato** [davesteele/comitup#181] [General]
When the normal WiFi connection is lost, is there a timeout value that I can change such that comitup does not transition to the startup/hotspot state until 'x' minutes?  this is to prevent temporary wifi outages from needing someone to reboot the device. Thanks  Maybe this is a new feature, not sure

**Comment by davesteele** [comment]
A system in HOTSPOT mode will retry CONNECTIONs every three minutes, unless someone is connected to the HOTSPOT.

There is a recently added feature that deletes a connection definition if the authentication fails - i.e. the user typed it in wrong. I've seen that code get triggered when my upstream router loses its mind. Is that your problem?

In either case, rebooting is unnecessary.

**Reply by mennamorato** [reply]
Thanks Dave for such quick responses. OK so now I just realized my Mac Mini connected to the hotspot and hence it did not reconnect to the wifi....  this will be tricky as I will have to tell people to 'forget' the wifi connection on their phone / laptop/whatever after setting it up but that works for me.

**Reply by davesteele** [reply]
Note that a device with two WiFi interfaces, in router mode, will make the connection attempt regardless of HOTSPOT connections. In that case you should have a password on the HOTSPOT.

**Reply by mennamorato** [reply]
Thanks I do not have this use case..using a Pi zero w
